[Console]::Error.WriteLine("Nothing to worry about.")
exit 1
